napari_clusters_plotter/_plotter.py

```diff
@@ -146,7 +147,21 @@ class MyNavigationToolbar(NavigationToolbar):
     def __init__(self, canvas, parent):
         super().__init__(canvas, parent)
         self.canvas = canvas
-
+    
+    def _update_buttons_checked(self):
+        super()._update_buttons_checked()
+        # changes pan/zoom icons depending on state (checked or not)
+        if 'pan' in self._actions:
+            if self._actions['pan'].isChecked():
+                self._actions['pan'].setIcon(QIcon("images//my_toolbar_icons//pan_checked.png"))
```

Comment:
```suggestion
                self._actions['pan'].setIcon(QIcon("images//my_toolbar_icons//Pan_checked.png"))
```
Should this maybe be a capital letter since file names are starting with capital letters, or it doesn't matter? This didn't fix the problem that I still don't see the icons though

Reply:
It could contribute to the problem, but it's not its main cause. Thanks for noticing that, I updated it in the next commit 😃